design/design-foundry_image_guidelines.md

```diff
@@ -0,0 +1,191 @@
+
+![](https://paper-attachments.dropbox.com/s_9EDE4747C43217AD4AEE83786B931C7D66AD1C729C8D13D539A157A9E65DAADB_1560350441871_Screen+Shot+2019-06-12+at+10.40.26+AM.png)
+
+### Examples
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_9EDE4747C43217AD4AEE83786B931C7D66AD1C729C8D13D539A157A9E65DAADB_1548349316084_Screen+Shot+2019-01-24+at+12.01.37+PM.png)
+
+
+
+## Case Study Images
+
+A case study image will belong in one of the four Foundry categories: UX & Content, Design, Development, or Culture. This is because a case study typically focuses on one of these areas. Identify the appropriate case study layer in the starter file that contains the gradient stroke for that category. Place the client’s logo centered in the frame. Be sure to use ample white space around the logo.
+
+
+![This is a case study that belongs in the  Design category.](https://paper-attachments.dropbox.com/s_9EDE4747C43217AD4AEE83786B931C7D66AD1C729C8D13D539A157A9E65DAADB_1560351236090_case-study_boom-crate.jpg)
+This is a case study that belongs in the Design category.
+
+![This is a case study that belongs in  the Development category.](https://paper-attachments.dropbox.com/s_9EDE4747C43217AD4AEE83786B931C7D66AD1C729C8D13D539A157A9E65DAADB_1560351250646_case-study_dealer-tire-01.jpg)
+This is a case study that belongs in the Development category.
+
+
+
+## Meet the Team Images
+
+Meet the team images are used with Foundry posts that introduce new team members. They exist in the Culture category, which uses blue from the color palette. When creating these images, open the starter file…
+
+https://www.dropbox.com/s/aywolmsfn1pmrm4/Foundry%20Team%20Image.psd?dl=0
+
+
+Drop in the person’s photo and run this Photoshop action:
+
+https://www.dropbox.com/s/xxw877dh8u3o9lj/Foundry%20Culture%20Images.atn?dl=0
+
+
+After running the action, it should look like these:
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_9EDE4747C43217AD4AEE83786B931C7D66AD1C729C8D13D539A157A9E65DAADB_1548354049564_Illustration-Test_182x.png)
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_9EDE4747C43217AD4AEE83786B931C7D66AD1C729C8D13D539A157A9E65DAADB_1548354037109_Illustration-Test_192x.png)
+
+
+
+## Announcement Images
+
+The announcement post image is used for Foundry posts that contain announcements like a job opening, appearance on a podcast, call for apprentice applications, workshop promotion, etc. Similar to case study images, announcement images could apply to any category. Announcement posts generally contain a title and Sparkbox logo. They can also optionally contain subtext which can serve to give more context. There are a few different types and variations of announcement images explained below:
+
+![This is an announcement image in the  Design Category.](https://paper-attachments.dropbox.com/s_9EDE4747C43217AD4AEE83786B931C7D66AD1C729C8D13D539A157A9E65DAADB_1560351763090_announcement-design-systems-survey.jpg)
```

Comment:
Would it be beneficial to link the keynote template here that lets us create these announcement images?

Reply:
Good call, added.